packages/react-next/src/components/TeachingBubble/TeachingBubble.types.ts

```diff
@@ -21,9 +19,7 @@ export interface ITeachingBubble {
  * TeachingBubble component props.
  * {@docCategory TeachingBubble}
  */
-export interface ITeachingBubbleProps
-  extends React.ClassAttributes<TeachingBubbleBase | TeachingBubbleContentBase>,
-    IAccessiblePopupProps {
+export interface ITeachingBubbleProps extends React.ClassAttributes<HTMLElement>, IAccessiblePopupProps {
```

Comment:
This isn't a class component, so having it continue extending `React.ClassAttributes` seems wrong. You should probably switch it to either `React.RefAttributes` or `React.Attributes`.